Assets/Scripts/ResultScript.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class ResultScript : MonoBehaviour
+{
+    bool next = false;
+    [SerializeField] GameObject result_ui;
+    [SerializeField] GameObject next_ui;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Return) && !next){
```

Comment:
```suggestion
        if (Input.GetKeyDown(KeyCode.Return) && !next)
        {
```
コード表記のズレです。エディタフォーマットの設定をVisualStudio(使用しているIDE)で設定しておくと便利です。